### DISCUSSION TITLE
Is the `variant="link"` for `Button` no longer meant to have a underline text?

### DISCUSSION THREAD

**Discussion by GollyJer** [GeekyAnts/NativeBase#4965] [Q&A] [answered]
It seems some of the button behavior doesn't match what's in the nativebase docs.

Examples... 
`variant="outline"` has a gray border instead of colored.
`variant="link"` doesn't have underlined text.

Are these by design and just not documented yet? Thanks!

**Comment by ankit-tailor** [comment, marked as answer]
Hey @GollyJer, These are new design changes in [v3.4.0](https://github.com/GeekyAnts/NativeBase/releases/tag/v3.4.0).

**Reply by GollyJer** [reply]
Cool. I see the docs have been updated also.
https://docs.nativebase.io/button#h3-variants
👍
